app/src/main/res/values-cat/help.xml

```diff
@@ -109,50 +109,6 @@
         Aquest és el bloc de la cadena de Monero on la vostra transacció ha estat inclosa.
     ]]></string>
 
-    <string name="help_send"><![CDATA[
```

Comment:
I mean I can just go over and just cherry-pick the barebones, but the reason why I went over with this is that we can't for sure know for every language that they are in fact "good translations"
Changing wording can mean very different things in different languages and affect what gender's used, word position, stuff like that, that's the reason I went over like this, the people who want to make these translations work out will and with Weblate any changes in the future can be easily fixed by translators and not go unnoticed

If you're fine with that, I'll do it

Reply:
Weblate has a pretty nice graceful way to deal with "fuzzy" translations, they're marked as needing for editing and show the differences done in the source string, but it doesn't do anything about old commits before integration with it, that's basically what I tried to achieve here, in the end it'd be less work in the end to you and the translators IMO
A lot of translators aren't gonna have the energy/time/want to go through everything, and just wanna be able to translate right off the bat, I am like that for certain apps, especially when they have a messy situation anyway

RN my suggestion is: check #975 and I'll rebase this and then you can see if you want this, so that we can get weblate hosting at least since we'll need those to get things working at a basic level